pkg/band/ism_2400.go

```diff
@@ -0,0 +1,147 @@
+		{
+			Frequency:   2403000000,
+			MaxDataRate: ttnpb.DATA_RATE_7,
+		},
+		{
+			Frequency:   2425000000,
+			MaxDataRate: ttnpb.DATA_RATE_7,
+		},
+		{
+			Frequency:   2479000000,
+			MaxDataRate: ttnpb.DATA_RATE_7,
+		},
+	}
+	const beaconFrequency = 2422000000
+
+	downlinkDRTable := [8][6]ttnpb.DataRateIndex{
+		{0, 0, 0, 0, 0, 0},
+		{1, 0, 0, 0, 0, 0},
+		{2, 1, 0, 0, 0, 0},
+		{3, 2, 1, 0, 0, 0},
+		{4, 3, 2, 1, 0, 0},
+		{5, 4, 3, 2, 1, 0},
+		{6, 5, 4, 3, 2, 1},
+		{7, 6, 5, 4, 3, 2},
+	}
+
+	ism_2400 = Band{
+		ID: ISM_2400,
+
+		MaxUplinkChannels: 16,
+		UplinkChannels:    defaultChannels,
+
+		MaxDownlinkChannels: 16,
+		DownlinkChannels:    defaultChannels,
+
+		SubBands: []SubBandParameters{
+			{
+				MinFrequency: 2400000000,
+				MaxFrequency: 2500000000,
+				DutyCycle:    1,
+				MaxEIRP:      8.0 + eirpDelta,
+			},
+		},
+
+		DataRates: map[ttnpb.DataRateIndex]DataRate{
+			ttnpb.DATA_RATE_0: makeLoRaDataRate(12, 812000, makeConstMaxMACPayloadSizeFunc(59)),
+			ttnpb.DATA_RATE_1: makeLoRaDataRate(11, 812000, makeConstMaxMACPayloadSizeFunc(123)),
+			ttnpb.DATA_RATE_2: makeLoRaDataRate(10, 812000, makeConstMaxMACPayloadSizeFunc(230)),
+			ttnpb.DATA_RATE_3: makeLoRaDataRate(9, 812000, makeConstMaxMACPayloadSizeFunc(230)),
+			ttnpb.DATA_RATE_4: makeLoRaDataRate(8, 812000, makeConstMaxMACPayloadSizeFunc(230)),
+			ttnpb.DATA_RATE_5: makeLoRaDataRate(7, 812000, makeConstMaxMACPayloadSizeFunc(230)),
+			ttnpb.DATA_RATE_6: makeLoRaDataRate(6, 812000, makeConstMaxMACPayloadSizeFunc(230)),
+			ttnpb.DATA_RATE_7: makeLoRaDataRate(5, 812000, makeConstMaxMACPayloadSizeFunc(230)),
+		},
+		MaxADRDataRateIndex: ttnpb.DATA_RATE_5,
+
+		ReceiveDelay1:    defaultReceiveDelay1,
+		ReceiveDelay2:    defaultReceiveDelay2,
+		JoinAcceptDelay1: defaultJoinAcceptDelay1,
+		JoinAcceptDelay2: defaultJoinAcceptDelay2,
+		MaxFCntGap:       defaultMaxFCntGap,
+		ADRAckLimit:      defaultADRAckLimit,
+		ADRAckDelay:      defaultADRAckDelay,
+		MinAckTimeout:    defaultAckTimeout - defaultAckTimeoutMargin,
+		MaxAckTimeout:    defaultAckTimeout + defaultAckTimeoutMargin,
+
+		DefaultMaxEIRP: 16,
+		TxOffset: []float32{
+			0,
+			-2,
+			-4,
+			-6,
+			-8,
+			-10,
+			-12,
+			-14,
+		},
+
+		Rx1Channel: channelIndexIdentity,
+		Rx1DataRate: func(idx ttnpb.DataRateIndex, offset uint32, _ bool) (ttnpb.DataRateIndex, error) {
+			if idx > ttnpb.DATA_RATE_7 {
+				return 0, errDataRateIndexTooHigh.WithAttributes("max", 7)
+			}
+			if offset > 5 {
+				return 0, errDataRateOffsetTooHigh.WithAttributes("max", 5)
+			}
+			return downlinkDRTable[idx][offset], nil
+		},
+
+		GenerateChMasks: generateChMask16,
+		ParseChMask:     parseChMask16,
+
+		LoRaCodingRate: "4/8",
+
+		FreqMultiplier:   200,
+		ImplementsCFList: true,
+		CFListType:       ttnpb.CFListType_FREQUENCIES,
+
+		DefaultRx2Parameters: Rx2Parameters{ttnpb.DATA_RATE_0, 2422000000},
+
+		Beacon: Beacon{
+			DataRateIndex:    ttnpb.DATA_RATE_3,
+			CodingRate:       "4/8",
```

Comment:
```suggestion
			CodingRate:       "4/5",
```

Reply:
Source?

Reply:
It is `4/8` all the time

Reply:
From the spec linked in https://github.com/TheThingsIndustries/lorawan-stack/issues/1649
![2020-04-29-12:27:15-screenshot](https://user-images.githubusercontent.com/12877905/80586153-d2de8980-8a14-11ea-94e2-101bc981d8f8.png)